### PR TITLE
Add PostgreSQL 13.3 commodity

### PR DESCRIPTION
**Pull request by IWHarvey**
* **What kind of change does this PR introduce (Bug fix, feature, docs update, ...)?**
  _Adds support for PostgreSQL 13.3._

* **What is the current behavior?**
  _It only has commodity support for PostgreSQL 9.4 and 9.6._

* **What is the new behavior (if this is a feature change)?**
  _Adds a PostgreSQL 13.3 commodity which can be specified using a "postgres-13" entry in commodidies list in an application's configuration.yml._

* **Does this PR introduce a breaking change? If so, what actions will users need to take in order to regain compatibility?**
  _No breaking changes - existing PostgreSQL 9.4/9.6 commodities are unaffected._

* **Notes**
  _This uses the postgres:13.3 image (consistent with other commodities using e.g. postgres:9.6). This image has PostGIS 3.1.2 available in the repositories. RDS ships PostGIS 3.0.3 with PostgreSQL 13.3, so there's a version mismatch here.
Needs further testing to see if this is going to cause issues._

### All submissions

* [x] Have you followed the guidelines in our [Contributing](CONTRIBUTING.md) document?
* [x] Is this PR targeting the `develop` branch, and the branch rebased with commits squashed if needed?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### New feature and bug fix submissions

* [x] Has your submission been tested locally?
* [x] Has documentation such as the [README](/README.md) been updated if necessary?
* [x] Have you linted your new code (using rubocop and markdownlint), and are not introducing any new offenses?

### CODE REVIEW

**Comment by matthew-shaw** [comment]
Wondering if this should either:

- Add deprecation warnings when provisioning a postgres 9.4 commodity, and plan to remove 9.4 entirely after 9.6 is deprecated in November 2021. (non-breaking change)

or

- Remove 9.4 now, since we are already 18 months past it's final release. (breaking change)

Then, add the same deprecation warning to 9.6 in November 2021, following this [version schedule](https://www.postgresql.org/support/versioning/), encouring existing and new apps to use a 13.x.x version.

File: scripts/provision_alembic.rb [hunk 1]
```diff
@@ -29,25 +29,35 @@ def provision_alembic(root_loc)
...
   app_configuration = YAML.load_file("#{root_loc}/apps/#{appname}/configuration.yml")
   do_migration = app_configuration.fetch('perform_alembic_migration', true)
   puts colorize_yellow("Dev-env-triggered Alembic migration disabled for #{appname}, skipping") if do_migration == false
   do_migration
 end
 
-def start_postgres_for_alembic
-  run_command_noshell(ENV['DC_CMD'].split(' ') + ['up', '-d', 'postgres'])
+def start_postgres_for_alembic(postgres_version)
+  container = postgres_container(postgres_version)
```
Comment (sichapman) [inline scripts/provision_alembic.rb:47]: This is not defined or imported

File: scripts/provision_postgres.rb [hunk 2]
```diff
@@ -72,10 +109,53 @@ def start_postgres(root_loc, appname, started)
...
+
+def show_postgres_94_warning()
+  puts colorize_yellow('*******************************************************')
+  puts colorize_yellow('**                                                   **')
+  puts colorize_yellow('**                     WARNING!                      **')
+  puts colorize_yellow('**                                                   **')
+  puts colorize_yellow('**         POSTGRESQL 9.4 IS OUT OF SUPPPORT         **')
+  puts colorize_yellow('**                                                   **')
+  puts colorize_yellow('** PostgreSQL 9.4 is out of support and no longer    **')
+  puts colorize_yellow('** available in higher environments. Please update   **')
```
Comment (sichapman) [inline scripts/provision_postgres.rb:142]: This is very HMLR-specific

File: scripts/provision_alembic.rb (hunk 1 above)
Comment (IWHarvey) [reply scripts/provision_alembic.rb:48]: Explicit requires statement added

File: scripts/provision_postgres.rb (hunk 2 above)
Comment (IWHarvey) [reply scripts/provision_postgres.rb]: Reworded more generically.

File: scripts/docker/postgres-13/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
+FROM postgres:13.3
```
Comment (sichapman) [inline scripts/docker/postgres-13/Dockerfile:1]:
```suggestion
FROM postgres:13.4
```

(Checked with AM)

File: scripts/provision_postgres.rb [hunk 4]
```diff
@@ -1,69 +1,106 @@
...
   config = YAML.load_file("#{root_loc}/dev-env-config/configuration.yml")
   return unless config['applications']
 
   # Did the container previously exist, if not then we MUST provision regardless of .commodities value
   new_db_container = false
-  if new_containers.include?('postgres')
+  if new_containers.include?(container)
     new_db_container = true
-    puts colorize_yellow('The Postgres container has been newly created - '\
+    puts colorize_yellow("The Postgre #{postgres_version} container has been newly created - "\
```
Comment (sichapman) [inline scripts/provision_postgres.rb:32]: s

File: scripts/docker/postgres-13/Dockerfile (hunk 3 above)
Comment (IWHarvey) [reply scripts/docker/postgres-13/Dockerfile]: Updated

File: scripts/provision_postgres.rb (hunk 4 above)
Comment (IWHarvey) [reply scripts/provision_postgres.rb]: Fixed

**Pull request merged**